package.json

```diff
@@ -18,8 +18,8 @@
     "transfercontext": "node ./packages/kolibri-tools/lib/i18nSyncContext.js run && yarn lint-frontend:format",
     "watch": "kolibri-tools build dev --file ./build_tools/build_plugins.txt",
     "watch-hot": "yarn run watch --hot",
-    "django-devserver": "kolibri manage --debug runserver --settings=kolibri.deployment.default.settings.dev \"0.0.0.0:8000\" ",
-    "services": "kolibri services --foreground --skip-update --port=8000",
+    "django-devserver": "kolibri manage --debug --skip-update runserver --settings=kolibri.deployment.default.settings.dev \"0.0.0.0:8000\" ",
```

Comment:
why is skip-update set here? i.e. when starting with yarn django-devserver on a new directory, the database will have no tables created. What is the rationale behind it?

Reply:
It's to prevent both the devserver and the services from running the update - but I might be able to pass this in when they are both run together, will check.